scormxblock/static/js/src/scormxblock.js

```diff
@@ -124,7 +127,20 @@ function ScormXBlock_${block_id}(runtime, element) {
       }
       return false;
     }
+    function disableNextModuleArrow() {
+      var next_link = document.getElementsByClassName("next")[0];
```

Comment:
This does not work for new UI. 
Secondly, Is using jquery css selector `$('.controls .next')` a more reliable solution?